h/resources.py

```diff
@@ -140,6 +140,42 @@ def __getitem__(self, pubid):
         return organization.logo
 
 
+class GroupResource(object):
+    def __init__(self, group, request):
+        self.request = request
```

Comment:
It is my hope that in the long term, these Resources won't need to gobble up an entire `request` object, but for the short term until we re-factor the way we're generating links it's significantly less painful to just toss the `request` in here. It feels less gross in the `Resources` module—I mean, this is very pyramid-specific so `request` doesn't feel as out of line.